File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.cs
using System;

public class Solution {
    public string solution(int age) {
        char[] charArray = age.ToString().ToCharArray();
        
        for(int i = 0; i < charArray.Length; i++)
        {
            int newChar = (int)char.GetNumericValue(charArray[i]) + 'a';
            charArray[i] = (char)newChar;
        }
        
        return string.Concat(charArray);
    }
}

File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int age) {
    string answer = "";
    
    while (age != 0)
    {
        int n = age % 10;
        answer = (char)('a' + n) + answer;
        age /= 10;
    }
    
    return answer;
}